app/src/src_nonfreenet/org/breezyweather/sources/SourceManager.kt

```diff
@@ -125,6 +127,7 @@ class SourceManager @Inject constructor(
         chinaService,
         nwsService,
         bmkgService,
+        mgmService,
```

Comment:
Needs rebase. I reworked that file to sort national sources by alphabetical order

Reply:
Oh cool! I'll take a look.